tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/worker/WaitForAllActionsTest.java

```diff
@@ -243,9 +243,13 @@ public void waitTest17() {
         Assert.assertEquals(((BMap) returns[0]).getMap().values().size(), 2);
         Assert.assertEquals(((BMap) returns[0]).getMap().values().size(), 2);
         Assert.assertTrue(Arrays.asList("{f2:150, f1:7}", "{f1:7, f2:150}")
-                                .contains(((BMap) returns[0]).getMap().values().toArray()[0].toString()));
-        Assert.assertTrue(Arrays.asList("{\"name\":\"hello foo\", \"id\":12}", "{\"id\":\"12\", " +
-                "\"name\":\"hello foo\"}").contains(((BMap) returns[0]).getMap().values().toArray()[1].toString()));
+                .contains(((BMap) returns[0]).getMap().values().toArray()[0].toString()));
+
+        String mapString = ((BMap) returns[0]).getMap().values().toArray()[1].toString();
+        if (!Arrays.asList("{\"name\":\"hello foo\", \"id\":12}", "{\"id\":\"12\", \"name\":\"hello foo\"}")
+                .contains(mapString)) {
+            Assert.fail("Wrong output: " + mapString);
```

Comment:
If this test failure is rare, is it better if we also add a message like "inform runtime team if this test fails with the error output", something like that so that others will let us know about this, even after the `runtime-swanlake-update-fixes` branch is merged to the `master` branch.

Reply:
Added the fix in https://github.com/ballerina-platform/ballerina-lang/pull/34690